platform/src/test/java/org/hillview/test/EqualityFilterTest.java

```diff
@@ -17,6 +17,7 @@
 
 package org.hillview.test;
 
+import org.hillview.dataset.LocalDataSet;
```

Comment:
Why is this needed now?

Reply:
Just a leftover of the debugging process. Removed.